### PR TITLE
redirecting to feed after login

### PR DESCRIPTION
**Pull request by jimkalis16**
changed it so it redirects you to feed after login i couldn't test but its seems like it would work just fine if i did :) feel free to edit or ask me to do it if anything is wrong :) 
Issue: [#356](https://github.com/LangCorrect/server/issues/356)

@keyxcode if you have time feel free to test it

### CODE REVIEW

**Comment by keyxcode** [comment]
Hi and thanks @jimkalis16 !

What you wrote looks very logical but unfortunately it's not working here for me because there's neither a `journals` namespace nor a `redirect` path. There's a `home` path though and I think we could use it for this @danielzeljko

File: config/settings/base.py [hunk 1]
```diff
@@ -139,7 +139,7 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#auth-user-model
 AUTH_USER_MODEL = "users.User"
 # https://docs.djangoproject.com/en/dev/ref/settings/#login-redirect-url
-LOGIN_REDIRECT_URL = "users:redirect"
+LOGIN_REDIRECT_URL = "journals:redirect"
```
Comment (danielzeljko) [inline config/settings/base.py:142]: This should be:

    LOGIN_REDIRECT_URL = "posts:list"

**Review by danielzeljko — CHANGES_REQUESTED** [review]

**Comment by danielzeljko** [comment]
So that home path would work because that view is checking if the user is logged in and redirecting to the correct place. I think in this case it would be better to take the user directly to the posts list: `posts:list`

File: config/settings/base.py (hunk 1 above)
Comment (jimkalis16) [reply config/settings/base.py]: thanks for the help :)

**Review by danielzeljko — APPROVED** [review]

**Pull request merged**